tests/contrib/test_multistore_file.py

```diff
@@ -186,9 +187,8 @@ def test_multistore_no_symbolic_link_files(self):
             'user-agent/1.0',
             ['some-scope', 'some-other-scope'])
         try:
-            self.assertRaises(
-                locked_file.CredentialsFileSymbolicLinkError,
-                store.get)
+            with self.assertRaises(CredentialsFileSymbolicLinkError):
```

Comment:
Was running over 80 characters here, so I had to adjust this.  I figured that breaking this `assertRaises` line into multiple lines could be a little confusing, so instead, I adjusted the way this is imported. (importing just the exception instead of the module)